src/components/movie-card.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { baseImageUrl } from "../services/instances";
+import { Box, Flex, Heading, Icon, Text } from "@chakra-ui/react";
+import { StarIcon } from "@chakra-ui/icons";
+
+export const MovieCard = (props) => {
+  console.log(props.image);
+  const bg = baseImageUrl + props.image;
+  return (
+    <>
+      <Box
+        color="white"
+        backgroundImage={`url(${bg})`}
+        bgPosition="top"
+        position="relative"
+        pt="160%"
+        backgroundSize="cover"
+        borderRadius={10}
+      >
+        <Box
+          position="absolute"
+          h="110px"
+          bottom="0"
+          left="0"
+          w="100%"
+          borderRadius={10}
+          bg="rgba(0,0,0,0.7)"
+        >
+          <Text color="#ffc000" pl={6} pt={4}>
+            {props.year.slice(0, 4)}
```

Comment:
This should probably get converted to a date first instead of slicing from a string. Something like this should work:
```js
new Date(props.year).getFullYear();
```